### PR TITLE
[STC6] remove mapped mass

### PR DESCRIPTION
**Pull request by damienmarchal**
______________________________________________________

This PR: 
- [ ] builds with SUCCESS for all platforms on the CI.
- [ ] does not generate new warnings.
- [ ] does not generate new unit test failures.
- [ ] does not generate new scene test failures.
- [ ] does not break API compatibility.
- [ ] is more than 1 week old (or has fast-merge label).

**Reviewers will merge only if all these checks are true.**

### CODE REVIEW

**Comment by damienmarchal** [comment]
@guparan Could we focus on having the stc5 & stc6 merged because it is very annoying to have worked and see no results.

**Comment by guparan** [comment]
[ci-build][with-scene-tests]

**Comment by guparan** [comment]
@damienmarchal Could you take a look at the 8 failing scene tests?
All of them are "required data" errors.

**Comment by damienmarchal** [comment]
Sure...

I split the PR by removing everything that was not related to the removable of mapped mass. But I still have no idea why the required data are failing know.

**Comment by damienmarchal** [comment]
Actually the failure are generated by error messages like: 
[ERROR]   [BeamFEMForceField(FEM)] Required data "poissonRatio" has not been set. (Current value is 0.49)

My first comment is that these message are 'wrong' because they are "errors" while there is a default value provided to the field. So because of these default value the fields shouldn't be taggued as "required".

File: SofaKernel/framework/sofa/core/objectmodel/BaseObject.cpp [hunk 1]
```diff
@@ -244,36 +238,22 @@ void BaseObject::copyAspect(int destAspect, int srcAspect)
...
 void BaseObject::init()
 {
-
-
 	for(VecData::const_iterator iData = this->m_vecData.begin(); iData != this->m_vecData.end(); ++iData)
 	{
 		if ((*iData)->isRequired() && !(*iData)->isSet())
 		{
-            serr << "Required data \"" << (*iData)->getName() << "\" has not been set. (Current value is " << (*iData)->getValueString() << ")" << sendl;
+            msg_error() << "Required data \"" << (*iData)->getName() << "\" has not been set. (Current value is " << (*iData)->getValueString() << ")" ;
```
Comment (damienmarchal) [inline SofaKernel/framework/sofa/core/objectmodel/BaseObject.cpp]: Here is the problematic line that cause scene errors. 
in previous code 'serr' is actually emitting a warning (I already told all the bad I think about that ;))
I replaced that with msg_error which is wrong.

**Comment by damienmarchal** [comment]
Done...can you review and merge :)

EDIT: actually I put everything in PR #831

**Pull request closed without merging**